scripts/report_auto_merge_deploy.py

```diff
@@ -33,13 +33,13 @@
             )
             deployment = github_api.get_deployment(sha)
             if deployment is None:
-                raise Exception("No deployment found")
+                deployment = github_api.create_deployment(sha)
             github_api.create_deployment_status(deployment["id"], "failure")
         elif state == "pending":
             github_api.create_commit_status(
                 sha,
                 "pending",
-                "Building images",
+                "Requested deployment",
```

Comment:
It's a thing that sends status updates on open pull requests and adds the "deployed to review.staging.kitspace.dev" notifications. 

> Will this change how deployment to the staging servers works?

Not this PR, I had a bunch of stuff that already went in since I needed to merge to test properly. Basically all PRs ready for review get merged into the "review" branch and get deployed. I should add some words to the readme about it.

Reply:
Added some docs about it: https://github.com/kitspace/kitspace-v2#auto-deploys